src/index.html

```diff
@@ -262,6 +262,125 @@ <h3 class="text-xl text-gray-50 font-semibold">
   </div>
 </section>
 
+<!-- Integrations -->
+<section id="integrations">
+  <div class="m-auto max-w-7xl px-4 py-12 md:pt-20 flex flex-col items-center">
```

Comment:
max-x-7xl is unused:

```suggestion
  <div class="m-auto px-4 py-12 md:pt-20 flex flex-col items-center">
```